config/traits.yml

```diff
@@ -786,6 +786,21 @@ traits:
       description: Months during which taxon has open (anthetic) flowers; keyed as a sequences of 12 Ns (not flowering) and Ys (flowering) starting with January
       type: categorical
       label: Range of flowering period
+    flowering_cues:
```

Comment:
For this new `flowering_cues` trait, is it intended for the allowed values to include details about rain timing, instead of being split off into its own trait, i.e. `post_rain_flowering`, like we did with `post_fire_flowering`? 
Another difference with the `post_fire_flowering` trait is that the timing of fire was input as a context variable, not within the trait value itself. Would it be worth choosing one system and keeping it consistent?

Reply:
I've thought about this for a while and I think I'm going to leave it. I think for rainfall it must be an evolved response to a combination of season and moisture that we're scoring. And then we can talk about the fire trait in the upcoming workshop.